data-reporting-service/organization-service/src/main/java/gov/cdc/etldatapipeline/organization/model/dto/orgdetails/Phone.java

```diff
@@ -0,0 +1,45 @@
+package gov.cdc.etldatapipeline.organization.model.dto.orgdetails;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrgElastic;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrgReporting;
+import lombok.*;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@ToString(callSuper = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class Phone implements OrgExtension {
+    private Long phTlUid;
+    private String phElpCd;
+    private String phElpUseCd;
+    private String telephoneNbr;
+    private String extensionTxt;
+    private String emailAddress;
+    private String phone_comments;
+
+    public <T> T updateOrg(T org) {
```

Comment:
I'll leave the ELP codes for ref
ph_elp_cd = 'PH'
ph_elp_use_cd = 'WP'

elp class_cd = 'TELE' is not sent in the json.

Reply:
Updated sproc